README.tmpl.md

```diff
@@ -125,6 +125,7 @@ The following companies/products use golangci-lint:
 * [Red Hat OpenShift](https://github.com/openshift/telemeter)
 * [Yahoo](https://github.com/yahoo/yfuzz)
 * [IBM](https://github.com/ibm-developer/ibm-cloud-env-golang)
+* [Intuit](https://github.com/argoproj)
```

Comment:
It probably makes sense to say `Argo Project` rather than `Intuit` since that's what it links to.

Reply:
interesting one - should we list both? we use it at Intuit on our OSS projects, which are published in various places including https://github.com/intuit

Reply:
Adding both links would be fine. In general, I think it's important for the descriptions to be precise about what they link to.